.remarkrc.yaml

```diff
@@ -0,0 +1,15 @@
+plugins:
+  # make remark aware of fontmatter.
+  remark-frontmatter:
+  # sensible linter defaults see: https://github.com/remarkjs/remark-lint/tree/main/packages/remark-preset-lint-recommended#rules
```

Comment:
This is where we configure what markdown linting rules we want. we start with a preset with a bunch of rules enabled, then add some of our own, then remove some of the preset rules that were too noisey